ReactWindows/ReactNative.Shared.Tests/Bridge/JavaScriptModuleRegistryTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Newtonsoft.Json.Linq;
+using NMock;
+using NMock.Syntax;
+using NUnit.Framework;
+using ReactNative.Bridge;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Events;
+using System.Threading;
+
+namespace ReactNative.Tests.Bridge
+{
+    [TestFixture]
```

Comment:
>TestFixture [](start = 5, length = 11)

When you open the project in Visual Studio 2015, do all of the tests still show up in the Test Explorer?  I.e., do both NUnit tests and MSTest tests show up together (when you have the NUnit plugin installed)?